jenkins/scripts/integration_test.sh

```diff
@@ -39,6 +41,7 @@ TESTS_FOR="${TESTS_FOR:-integration_test}"
 DEFAULT_HOSTS_MEMORY="${DEFAULT_HOSTS_MEMORY:-4096}"
 
 VM_TIMELABEL="${VM_TIMELABEL:-$(date '+%Y%m%d%H%M%S')}"
+TEST_EXECUTER_VOLUME_NAME="${TEST_EXECUTER_VOLUME_NAME:-ci-test-volume-${VM_TIMELABEL}}"
```

Comment:
that variable should be defined in the pipeline to not default to unpredictable name : https://github.com/metal3-io/project-infra/blob/master/jenkins/jobs/integration_tests.pipeline#L49. you can also directly use the vm name, instead of having another name for the volume

Reply:
Directly using the vm name